src/js/constants/countEntities.tsx

```diff
@@ -0,0 +1,14 @@
+import type { ReactNode } from 'react';
+import { ExperimentOutlined, FileOutlined, TeamOutlined } from '@ant-design/icons';
+import { BiDna } from 'react-icons/bi';
+
+import type { BentoCountEntity } from '@/types/entities';
+
+export const COUNT_ENTITY_REGISTRY: Record<BentoCountEntity, { icon: ReactNode }> = {
```

Comment:
thanks for making a central place to store this data

Reply:
Both are O(n) to say, it just design wise provides an easy O(1) access store if someone decided they wanted a count or not all the counts. So I prefer this style as we get one true data source and then a view selector.